docs/reference/api/docker_remote_api.md

```diff
@@ -64,12 +64,20 @@ without protocol. Throughout this structure, double quotes are required.
 
 ## Using Docker Machine with the API
 
-If you are using `docker-machine`, the Docker daemon is on a virtual host that uses an encrypted TCP socket. This means, for Docker Machine users, you need to add extra parameters to `curl` or `wget` when making test API requests, for example:
+If you are using `docker-machine`, the Docker daemon is on a virtual
+host that uses an encrypted TCP socket. This means, for Docker Machine
```

Comment:
I'm not sure the 'virtual host' part is *strictly* true (cf. the generic driver), but I don't think it makes any crucial difference in the following use of `curl` or `wget` besides `YOUR_VM`.